crates/frame-support-ex/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "pallet-common"
+name = "frame-support-ex"
```

Comment:
Please don't use `frame-support` because it's not a part of `frame`. Better to call it `phala-pallet-common` or something similar